### ISSUE TITLE
Fix time and view count display in list view

### ISSUE THREAD

**Issue by tbkissinger** [uchicago-sg/caravel#131]
Display the post date/time and view counts for all (currently the list view doesn’t show when a posting was posted)

**Comment by fatlotus** [comment]
There isn't a view count right now — just an inquiry count. Do you mean the latter, or would people also want the former?

**Comment by tbkissinger** [comment]
Yeah I posted this prior to seeing the inquiry count -- with that, no need
for a view count (except maybe for sellers, to know if they're getting
interest? But really not that necessary)
